create_pnor_image.pl

```diff
@@ -132,7 +137,7 @@
 $build_pnor_command .= " --binFile_VERSION $openpower_version_filename";
 $build_pnor_command .= " --binFile_IMA_CATALOG $scratch_dir/ima_catalog.bin.ecc";
 if ($release eq "p9"){
-    $build_pnor_command .= " --binFile_WOFDATA $scratch_dir/wofdata.bin.ecc";
+    $build_pnor_command .= " --binFile_WOFDATA $wofdata_binary_filename.ecc" if -e $wofdata_binary_filename.".ecc";
```

Comment:
Since all the changes are going in at different times, we wanted it so that we would pass if the input wasn't there. I'm not sure if we want to fail if it's not there right now.